includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -1048,6 +1049,42 @@ public function add_process_content_meta_to_rest_api() {
 		);
 	}
 
+	/**
+	 * Display a dismissable admin notice when a threshold still at the default 70% and IBM Watson NLU active
+	 */
+	public function thresholds_update_notice() {
+		$config = $this->is_configured();
+
+		$thresholds = [
+			$config['features']['category_threshold'] ?? 55,
+			$config['features']['keyword_threshold'] ?? 55,
+			$config['features']['entity_threshold'] ?? 55,
+			$config['features']['concept_threshold'] ?? 55,
+		];
+
+		if ( ! in_array( 70, $thresholds, true ) ) {
```

Comment:
In addition to my other note, there's a few issues with showing the message based on this condition:

- If someone has changed the threshold setting to something other than 70 (like 75 or 65), the message won't show
- If in the future they manually change the setting back to 70, the admin notice will always show for them

I think we need to consider an approach where we show a one-time admin notice the next time someone updates the plugin. We'll need to store something in an option to track that we showed that notice and not display it again